### ISSUE TITLE
Daily Build Suddenly Broken for Ubuntu Daily Release.

### ISSUE THREAD

**Issue by myungjoo** [nnstreamer/nnstreamer#2395]
http://nnsuite.mooo.com/nnstreamer/ci/taos/daily-build/build_result/2020-05-21/log/build_log_ubuntu_output_2020-05-21.txt

**Comment by myungjoo** [comment]
- Caffe2 @ NNStreamer-PPA should be rebuilt with protobuf 3.8.0
- TF-Lite @ NNStreamer-PPA needs to be verified with protobuf 3.8.0
- We can check potential issue of protobuf-3.8.0 in other platforms; we are going to update protobuf in Tizen as well!

**Comment by gichan-jang** [comment]
I tested with `object-detection-tf` of nnstreamer example. 
(the model requires version 3.8.0 of the Protocol Buffer runtime library.)
After installing protobuf 3.8.0 from nnstreamer ppa, it works well.
nnstreamer-examples using tf and tf-lite are tested.

**Comment by dsuess** [comment]
@gichan-jang Could you please give some more detail for that fix? I tried install `libprotobuf19` throught the PPA, but I am still getting the same error of mismatching protobuf versions

**Comment by gichan-jang** [comment]
@dsuess 
Sorry for late answer, I just checked your comment.
In short, We just reverted protobuf 3.8.0 to 3.6.1 and tensorflow 1.15.1 to 1.13.1
This issue has been relieved by @again4you . 

I explain the situation briefly :
1. nnstreamer ppa was updated to protobuf 3.8.0 and tensorflow 1.15.1
2. Tensorflow had no dependency problems with the updated version of protobuf.
3. Caffe2 and pytorch had problem with the protobuf 3.8.0.
4. The stable version of pytorch depends on protobuf 3.6.1
5. We determined to upgrade protobuf later.
